### PR TITLE
remove WalletContext - TransactionAuthorization UI bridging

### PR DESCRIPTION
**Pull request by ivan-zynesis**
reinstantiate jellyfish wallet for signing

#### What kind of PR is this?:

/kind refactor

#### What this PR does / why we need it:
Simplify passcode prompting implementation.
Reduce complexity, ease other party to understand and implement custom light wallet.

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: app/screens/TransactionAuthorization.tsx [hunk 1]
```diff
@@ -114,17 +122,29 @@ export function TransactionAuthorization (): JSX.Element | null {
...
         setAttemptsRemaining(MAX_PASSCODE_ATTEMPT - counter)
         emitEvent('IDLE')
       })
-      .catch((err) => Logging.error(err))
-  }, [])
+      .catch(e => {
+        Logging.error(e)
+        throw e // unexpected, unable to init
+      })
+  }, [walletFromContext])
```
Comment (ivan-zynesis) [inline app/screens/TransactionAuthorization.tsx:147]: Hook with original wallet context to reflect changes here.

File: app/screens/AppNavigator/screens/Dex/PoolSwap/ConfirmPoolSwapScreen.tsx [hunk 2]
```diff
@@ -113,14 +111,13 @@ export interface DexForm {
...
 async function constructSignedSwapAndSend (
-  account: WhaleWalletAccount, // must be both owner and recipient for simplicity
   dexForm: DexForm,
   dispatch: Dispatch<any>,
   postAction: () => void
 ): Promise<void> {
   try {
     const maxPrice = dexForm.fromAmount.div(dexForm.toAmount).decimalPlaces(8)
-    const signer = async (): Promise<CTransactionSegWit> => {
+    const signer = async (account: WhaleWalletAccount): Promise<CTransactionSegWit> => {
```
Comment (ivan-zynesis) [inline app/screens/AppNavigator/screens/Dex/PoolSwap/ConfirmPoolSwapScreen.tsx:120]: the main bug, should use walletAccount derived from TransactionAuthorization

File: cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ context('Wallet - Transaction Authorization with Error', () => {
 
   before(function () {
     cy.createEmptyWallet(true)
-    cy.sendDFItoWallet().sendTokenToWallet(['BTC']).wait(10000)
+    cy.sendDFItoWallet().sendTokenToWallet(['BTC']).wait(5000)
```
Comment (ivan-zynesis) [inline cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:12]: reduce wait time, DFI token transfer (without splitting into 2 actions in subsequent block) is automatically done in whale playground now.

File: cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts [hunk 4]
```diff
@@ -143,13 +143,13 @@ context('Wallet - Transaction Authorization', () => {
...
       Array.from(Array(4), (v, i) => i + 1).forEach(() => {
-        cy.getByTestID('pin_authorize').type('696969').wait(1000)
+        cy.getByTestID('pin_authorize').type('696969').wait(2000)
       })
       cy.url().should('include', 'wallet/onboarding')
     })
 
     it('should clear attempt on success', function () {
-      cy.createEmptyWallet(true)
+      cy.createEmptyWallet(true).wait(4000)
```
Comment (ivan-zynesis) [inline cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:152]: Spare time for encryption to run, probably too long, depend on hardware.

0 wait time randomly fail on my own machine.

File: cypress/integration/functional/wallet/dex/poolswap.spec.ts [hunk 5]
```diff
@@ -63,7 +63,7 @@ context('Wallet - DEX - Pool Swap with balance', () => {
   })
 
   it('should be able to click half', function () {
-    cy.getByTestID('50%_amount_button').click().wait(3000)
+    cy.getByTestID('50%_amount_button').click().wait(500)
```
Comment (ivan-zynesis) [inline cypress/integration/functional/wallet/dex/poolswap.spec.ts:66]: This is estimate output token amount, should not take long.

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**